### PR TITLE
Validation: description-missing fixes.

### PR DESCRIPTION
**Pull request by brittmsantos**
Added missing descriptions to fix validation errors.

### CODE REVIEW

File: BingMaps/v8-web-control/map-control-concepts/infoboxes/custom-html-infobox.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Custom HTML Infobox | Microsoft Docs"
+description: Provides a code example that shows how to create a custom infobox for pushpins using a HTML template string.
```
Comment (MeeraDi) [inline BingMaps/v8-web-control/map-control-concepts/infoboxes/custom-html-infobox.md:3]: an HTML template

File: BingMaps/v8-web-control/map-control-concepts/infoboxes/infobox-when-shape-clicked.md [hunk 2]
```diff
@@ -78,4 +79,4 @@ Often it is useful to be able to display an infobox when any `IPrimitive` shape;
 
 Running this code will display a polygon on the map. If you click on it, an infobox will appear filled in with the metadata for that polygon. 
 
-![BMV8_InfoboxShapeClicked](../../media/bmv8-infoboxshapeclicked2.png)
\ No newline at end of file
+![BMV8_InfoboxShapeClickedScreenshot of a Bing map showing an infobox associated with a polygon that has a description stating it is a polygon.](../../media/bmv8-infoboxshapeclicked2.png)
```
Comment (MeeraDi) [inline BingMaps/v8-web-control/map-control-concepts/infoboxes/infobox-when-shape-clicked.md:82]: Screenshot

File: BingMaps/v8-web-control/map-control-concepts/layers/basic-tile-layer-example.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Basic Tile Layer Example | Microsoft Docs"
+description: Provides a code example that shows how to set the set the `mercator` property to a TileSource that has a `uriConstructor` property set to a URL.
```
Comment (MeeraDi) [inline BingMaps/v8-web-control/map-control-concepts/layers/basic-tile-layer-example.md:3]: Provides a code example that shows how to set the

**Comment by brittmsantos** [comment]
@stevemunk @rbrundritt

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. Thank you!

File: BingMaps/v8-web-control/map-control-concepts/layers/data-layer-events.md [hunk 4]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Data Layer Events | Microsoft Docs"
+description: Provides a code example that shows how to add random shapes to a layer and attaches mouse events to the layer.
```
Comment (stevemunk) [inline BingMaps/v8-web-control/map-control-concepts/layers/data-layer-events.md:3]:
```suggestion
description: Example code demonstrating how to add a single event handler to a layer which will then apply to all shapes it contains. Compared to adding mouse events to shapes by looping through each shape and adding an event to it, this approach requires writing less code, reduces the amount of memory required and provides a small performance benefit.
```

File: BingMaps/v8-web-control/map-control-concepts/layers/index.md [hunk 5]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Layers | Microsoft Docs"
+description: Describes the Layer class, which separates multiple data sets as layers, and provides lists of code examples and references.
```
Comment (stevemunk) [inline BingMaps/v8-web-control/map-control-concepts/layers/index.md:3]:
```suggestion
description: The overview page for the Layers section contains links to examples and code references for the Layer, TileLayer and TileSource Classes and TileLayerOptions and TileSourceOptions Objects.
```

File: BingMaps/v8-web-control/map-control-concepts/map/check-birds-eye-coverage-availability.md [hunk 6]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Check Bird’s eye Coverage Availability | Microsoft Docs"
+description: Provides a code example that demonstrates the static function's ability to check if bird's eye imagery is available at a location.
```
Comment (stevemunk) [inline BingMaps/v8-web-control/map-control-concepts/map/check-birds-eye-coverage-availability.md:3]:
```suggestion
description: Provides a code example that demonstrates the static function's ability to check if bird's eye imagery is available at a location and then loads the map into bird’s eye or aerial mode accordingly.
```

File: BingMaps/v8-web-control/map-control-concepts/map/customize-map-options-on-load.md [hunk 7]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Customize Map Options on Load | Microsoft Docs"
+description: Provides a code example that customizes map options when the map is loaded to set it to an aerial map that is zoomed in over London, UK.
```
Comment (stevemunk) [inline BingMaps/v8-web-control/map-control-concepts/map/customize-map-options-on-load.md:3]:
```suggestion
description: Example code demonstrating how to change how the map is loaded in the GetMap function, setting the type to aerial, zoom level to 10, then centers the map over London, UK.
```

File: BingMaps/v8-web-control/map-control-concepts/map/index.md [hunk 8]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Map | Microsoft Docs"
+description: Describes the Map, which is the primary control in the Bing Maps V8 Map control interface, and provides examples and related topics.
```
Comment (stevemunk) [inline BingMaps/v8-web-control/map-control-concepts/map/index.md:3]:
```suggestion
description: The overview page for the Map section contains links to examples and code references including the Map Class and the MapOptions and ViewOptions Objects.
```

**Review by stevemunk — CHANGES_REQUESTED** [review]

**Comment by brittmsantos** [comment]
@stevemunk

I've implemented the requested changes. Please let me know if there are any other issues preventing approval and merge. Thank you, and have a great day!

File: BingMaps/v8-web-control/map-control-concepts/layers/basic-tile-layer-example.md [hunk 9]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Basic Tile Layer Example | Microsoft Docs"
+description: Provides a code example that shows how to set the `mercator` property to a TileSource that has a `uriConstructor` property set to a URL.
```
Comment (stevemunk) [inline BingMaps/v8-web-control/map-control-concepts/layers/basic-tile-layer-example.md:3]:
```suggestion
description: Provides a code example that shows how to set the mercator property to a TileSource that has a uriConstructor property set to a URL.
```

**Review by stevemunk — CHANGES_REQUESTED** [review]

**Comment by brittmsantos** [comment]
@stevemunk 

I've implemented the requested change. Please let me know if there are any other issues preventing approval and merge. Thanks!

File: BingMaps/v8-web-control/map-control-concepts/map/map-events.md [hunk 10]
```diff
@@ -1,6 +1,6 @@
 ---
-title: "Map Events Example | Microsoft Docs"
-description: Provides a code example that shows how different events for the map work by highlighting a label to indicate which event fired.
+title: "Map Events for V8 Web Control | Microsoft Docs"
+description: Learn about map events for V8 Web Control with an example showing how all the different events for the map work by highlight a label to indicate which event fired as you use the map.
```
Comment (stevemunk) [inline BingMaps/v8-web-control/map-control-concepts/map/map-events.md:3]:
```suggestion
description: Learn about map events for V8 Web Control with an example showing how all the different events for the map work by highlighting a label to indicate which event fired as you use the map.
```

**Pull request merged**